### DISCUSSION TITLE
Azure Pipeline Specify output path

### DISCUSSION THREAD

**Discussion by carlooospm** [allure-framework/allure2#2827] [Questions & Support]
I'm trying to specify the output path of the result file so that I can publish it as a downloadable artifact in a subsequent step, but there is no way, Is there an alternative, to download the file from azure pipeline? Like:

          - task: PublishAllureReport@1
            displayName: 'Generate & Publish Consolidated Allure Report'
            inputs:
              testResultsDir: '$(System.DefaultWorkingDirectory)/target/junit-results'
              reportName: "Consolidated Report"
              outputDir: '$(System.DefaultWorkingDirectory)/target/allure'----------------------------

          - task: PublishPipelineArtifact@1
            inputs:
              targetPath: '$(System.DefaultWorkingDirectory)/target/allure'
              artifactName: 'AllureReport-Consolidated'
              publishLocation: 'pipeline'
            displayName: 'Publish Consolidated Allure Report'

**Comment by epszaw** [comment]
Hey there! We can add a parameter to make possible to configure `outputPath` for the report, then you'll be able to download/publish report as an artifact in the following pipeline steps

**Reply by carlooospm** [reply]
That would be great! Let me know if it's on track!

Regards!

**Reply by slodusan-crayon** [reply]
Hi @epszaw I am also interested in this feature. Is it maybe already developed?
